app/controllers/offices_controller.rb

```diff
@@ -0,0 +1,27 @@
+class OfficesController < ApplicationController
+  def index
+    return unless search
+    return @error_message = search.error_message unless search.valid?
+
+    @transportation_offices = TransportationOffice.by_distance_with_shipping_office(search.result).paginate(page: params[:page])
+  end
+
+  private
+
+  def search
+    @search ||=
+      if params[:coordinates].present?
```

Comment:
I take it you decided not to have the zipcode search get transformed into coordinates params after all.

Reply:
@jamesatheyDDS Based on our IRL conversation from last week, we decided to punt on that for now.